### PR TITLE
deliveryom reports

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: src/main/resources/config/application.yml [hunk 1]
```diff
@@ -183,6 +183,7 @@ application:
       send-failed-queue-name: communication_failed_sms
       delivery-failed-queue-name: communication_delivery_failed
       delivered-queue-name: communication_delivered_reports
+      deliveredOmQueueName: communication_delivered_om_reports
```
Comment (amedvedchuk) [inline src/main/resources/config/application.yml]: different naming approaches mixed(kebab and camel cases)
```delivered-queue-name
deliveredOmQueueName```

File: src/main/java/com/icthh/xm/tmf/ms/communication/config/ApplicationProperties.java [hunk 2]
```diff
@@ -41,6 +41,7 @@
         private String sentQueueName;
         private String deliveryFailedQueueName;
         private String deliveredQueueName;
+        private String deliveredOmQueueName;
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/communication/config/ApplicationProperties.java]: what does it mean OM?

File: src/main/java/com/icthh/xm/tmf/ms/communication/messaging/SendToKafkaOmDeliveryReportListener.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.StopWatch;
+import org.jsmpp.bean.DeliverSm;
+import org.jsmpp.bean.MessageState;
+import org.jsmpp.bean.OptionalParameter;
+
+@Slf4j
+public class SendToKafkaOmDeliveryReportListener extends AbstractDeliveryReportListener {
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/communication/messaging/SendToKafkaOmDeliveryReportListener.java]: please add test

File: src/main/java/com/icthh/xm/tmf/ms/communication/config/ApplicationProperties.java (hunk 2 above)
Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/tmf/ms/communication/config/ApplicationProperties.java]: mobile originated

File: src/main/java/com/icthh/xm/tmf/ms/communication/config/MessagingConfiguration.java [hunk 4]
```diff
@@ -79,4 +80,16 @@ public SendToKafkaDeliveryReportListener deliveryReportListener(MessagingAdapter
                                                                             new LinkedBlockingQueue<>()));
     }
 
+    @Bean
+    public SendToKafkaMoDeliveryReportListener deliveryOmReportListener(MessagingAdapter messagingAdapter,
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/communication/config/MessagingConfiguration.java]: rename `deliveryOmReportListener` -> `deliveryMoReportListener`

**Pull request merged**